posthog/models/team.py

```diff
@@ -82,6 +83,16 @@ def get_default_data_attributes() -> Any:
     return ["data-attr"]
 
 
+@reversion.register(
+    exclude=(
+        "api_token",
+        "session_recording_retention_period_days",
+        "plugins_opt_in",
+        "opt_out_capture",
+        "event_names",
+        "event_properties",
+    )
```

Comment:
Should we add all deprecated fields here?

Reply:
Ha, yeah, same not seeing that one comment covers many lines :)